core/src/main/java/org/jdbi/v3/core/statement/BaseStatement.java

```diff
@@ -32,15 +31,6 @@
     BaseStatement(Handle handle) {
         this.handle = handle;
         this.ctx = new StatementContext(handle.getConfig().createCopy(), handle.getExtensionMethod());
-
-        // Prevent bogus signatures like Update extends SqlStatement<Query>
```

Comment:
This check was about 15% of overall time to execute a single statement :O

Reply:
Wow :grimacing: